src/main/java/calculator/StringAdder.java

```diff
@@ -0,0 +1,52 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAdder {
+    public static final int ZERO = 0;
+    public static String regex = ",|:";
+
+    public static int splitAndSum(String text) {
+        if (isBlank(text)) return ZERO;
+        if (isContainsNegative(text)) throw new RuntimeException();
+        return sum(split(text));
+    }
```

Comment:
```suggestion
    public static int splitAndSum(Number text) {
...
```

```
        if (isBlank(text)) return ZERO;
        if (isContainsNegative(text)) throw new RuntimeException();
```
조건이 이미 검증된 객체가 들어온다면 더 좋을것 같아요!